### ISSUE TITLE
mortar yank to ignore some resources

### ISSUE THREAD

**Issue by matti** [kontena/mortar#64]
When developing it just handy sometimes to fire+yank in a loop - if you happen to have let'sencrypt certs it is easy to get banned from let's encrypt because you constantly request a cert.

Would be handy to define "do not delete these resources" for yank - or something else.

**Comment by jnummelin** [comment]
> if you happen to have let'sencrypt certs it is easy to get banned from let's encrypt because you constantly request a cert.

Not sure why this would happen if there's no changes to the resource (a `certmanager.k8s.io/v1alpha1 Certificate` I presume). Mortar should not re-deploy the cert "request".

But I can see other possible cases for ignoring some resources in the yank phase.

**Comment by jnummelin** [comment]
you are right, of course the cert is yanked too. I had some mis-reading of the issue :)
